packages/twenty-chrome-extension/src/options/modules/ui/theme/constants/font.ts

```diff
@@ -0,0 +1,45 @@
+import { color, grayScale } from './colors';
```

Comment:
this will need to be refactored once we extract twenty-ui into it's own package

Reply:
I had a conversation with Felix about “twenty-ui” back in December, but it was not among the top priorities at that time. Therefore, I replicated this code. However, when you start extracting things out, I’d be happy to contribute to the UI library and refactor the code to utilize it - not a problem.